### PR TITLE
Implement practical example of Mender-configure Apply script in demo layers/onboarding

### PR DESCRIPTION
**Pull request by kacf**
Best reviewed commit by commit.

Note that this is going to fail because `jq` is missing the test image. This is a chicken and egg problem with https://github.com/mendersoftware/meta-mender/pull/1311. Probably we should just merge this PR first, assuming there are no other errors, and then retest once https://github.com/mendersoftware/meta-mender/pull/1311 passes successfully.

### CODE REVIEW

File: demo/mender-demo-raspberrypi-led [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+if [ $return_code -ne 0 ]; then
+    cat 1>&2 <<EOF
+Applying the led configuration returned $return_code. Invalid configuration?
+The available choices are:
+EOF
+    # Our own choices, just shortcuts.
+    echo -n "on off "
+    cat "$LED" 1>&2
+fi
+exit $return_code
```
Comment (lluiscampos) [inline demo/mender-demo-raspberrypi-led:84]: Maybe couple of newlines:
```suggestion
if ! [ -e "$LED" ]; then
    echo "$LED does not exist. Cannot apply LED configuration." 1>&2
    exit 1
fi

return_code=0
echo "$LED_CONF" > "$LED" || return_code=$?
echo "$LED_CONF_BRIGHTNESS" > "$LED_BRIGHTNESS" || return_code=$?
if [ $return_code -ne 0 ]; then
    cat 1>&2 <<EOF
Applying the led configuration returned $return_code. Invalid configuration?
The available choices are:
EOF
    # Our own choices, just shortcuts.
    echo -n "on off "
    cat "$LED" 1>&2
fi

exit $return_code
```

**Review by lluiscampos — APPROVED** [review]
Absolutely nothing to point to... good work!

**Comment by lluiscampos** [comment]
> Probably we should just merge this PR first, assuming there are no other errors, and then retest once [mendersoftware/meta-mender#1311](https://github.com/mendersoftware/meta-mender/pull/1311) passes successfully.

I agree with this process :+1:

**Review by tranchitella — APPROVED** [review]
AMAZING, simply perfect.

**Comment by kacf** [comment]
Good points, added more space and fixed Black formatting.

**Comment by kacf** [comment]
> Note that this is going to fail because `jq` is missing the test image. This is a chicken and egg problem with [mendersoftware/meta-mender#1311](https://github.com/mendersoftware/meta-mender/pull/1311). Probably we should just merge this PR first, assuming there are no other errors, and then retest once [mendersoftware/meta-mender#1311](https://github.com/mendersoftware/meta-mender/pull/1311) passes successfully.

Confirmed, this is [passing](https://gitlab.com/Northern.tech/Mender/mender-configure-module/-/pipelines/263901901) after https://github.com/mendersoftware/meta-mender/pull/1311 was merged.

**Pull request merged**